### ISSUE TITLE
Disable VirtualBox default DHCP

### ISSUE THREAD

**Issue by lmakarov** [blinkreaction/boot2docker-vagrant#16]
The issues and a workaround were pointed out in https://github.com/blinkreaction/drude/issues/20

This may require altering the base box packer template, doing this manually (as described in https://github.com/blinkreaction/drude/issues/20) or firing up a vboxmanage command in setup.sh

**Comment by lmakarov** [comment]
Disable (remove) the default DHCP server via command line

```
VBoxManage dhcpserver remove --netname HostInterfaceNetworking-vboxnet0
```

**Comment by lmakarov** [comment]
Another option is to kill the entire adapter the default DHCP server is attached to (vboxnet0)

```
VBoxManage hostonlyif remove vboxnet0
```

The adapter will be recreated when a machine restarts anyway.

**Comment by lmakarov** [comment]
On Windows

```
VBoxManage dhcpserver remove --netname "HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter"
VBoxManage hostonlyif remove "VirtualBox Host-Only Ethernet Adapter"
```

**Comment by lmakarov** [comment]
Released in [v1.1.0](https://github.com/blinkreaction/boot2docker-vagrant/releases/tag/v1.1.0)
